src/ui/atoms/fileInput/FileInput.tsx

```diff
@@ -75,6 +75,15 @@ export const FileInput: React.FC<FileInputProps> = ({
     event.stopPropagation()
   }, [])
 
+  const allowSelectSameFile = useCallback(
```

Comment:
IMHO, the naming should be more explicit
```suggestion
  const clearInputValue = useCallback(
```